server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const Hostels = require('../database/index.js');
+const mongoose = require('../database/index.js');
```

Comment:
What is the import on line 4 doing that's different than line 5? I think you meant to import `database/hostels.js` on line 4.